src/components/node-list/node-list-group.js

```diff
@@ -51,14 +55,77 @@ export const NodeListGroup = ({
         />
       </NodeListRow>
     </h3>
-
-    <ChildrenContainer
-      className={classnames(childrenClassName, 'pipeline-nodelist__children', {
-        'pipeline-nodelist__children--closed': collapsed
-      })}>
-      {children}
-    </ChildrenContainer>
-  </Container>
+    <LazyList
+      name={name}
+      height={(start, end) => (end - start) * nodeListRowHeight}
+      total={items.length}
+      onChange={({ start, end, total }) =>
+        console.log(
+          `${group.name} ${start} to ${end} (${end - start} of ${total})`
+        )
+      }>
```

Comment:
Adding a note about this to remind us both that it should be removed before merging

Might also be worth a comment in the code:`// @TODO remove this`

Reply:
Yeah it's in just to make this PR easier to test and review, once approved it will be removed.